src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -24,5 +27,6 @@ public void registerClasses(Kryo kryo) {
         // fix here.
         // We are tracking this issue with (#874)
         kryo.register(Collections.unmodifiableMap(Collections.EMPTY_MAP).getClass(), new UnmodifiableCollectionsSerializer());
+        kryo.register(AlignmentRecord.class, new AvroSerializer(ClassTag$.MODULE$.apply(AlignmentRecord.class)));
```

Comment:
I'm guessing it's ok to create a default constructor that instantiates an `ADAMKryoRegistrator` instance?